src/lightning_app/core/flow.py

```diff
@@ -274,12 +271,15 @@ def state_with_changes(self):
         }
 
     @property
-    def flows(self):
```

Comment:
You could keep this for compatibility and instead introduce `named_flows()`. This would make it consistent with named_works, which is also recursive.

Also, with this change, you can get rid of `get_all_children` which did the same.

Reply:
Removed !